pages/details/[counter].tsx

```diff
@@ -1,25 +1,25 @@
 import Head from 'next/head';
 import Link from 'next/link';
-import { GetStaticPaths, GetStaticProps } from 'next';
+import {GetStaticPaths, GetStaticProps} from 'next';
```

Comment:
I guess we have some linting issues here. I have no strong opinion on that matter, but it should be discussed in an other pull request, with a linter configuration and everything.

Can we skip for now reformating?

Reply:
Sure sorry about that, I'll update the PR

Reply:
After much fight against my IDE I finally found a way to revert my formatting changes..